### ISSUE TITLE
Add `deal` cheatcode

### ISSUE THREAD

**Issue by kasperski95** [software-mansion/protostar#180]
```solidity
function deal(address who, uint256 newBalance) external;
```

Sets the balance of an address who to newBalance.

```solidity
address alice = address(1);
cheats.deal(alice, 1 ether);
log_uint256(alice.balance); // 1000000000000000000
```

**Comment by MaksymilianDemitraszek** [comment]
Isn't this one addressed by #408

**Comment by kasperski95** [comment]
We may introduce predeployed accounts and this cheatcode could make sense. Currently, we don't sign e.g. internal declare tx in the declare cheatcode.
